certbot/certbot/_internal/cert_manager.py

```diff
@@ -90,9 +90,23 @@ def certificates(config):
 def delete(config):
     """Delete Certbot files associated with a certificate lineage."""
     certnames = get_certnames(config, "delete", allow_multiple=True)
+    disp = zope.component.getUtility(interfaces.IDisplay)
+    if len(certnames) > 1:
+        suffix = "s"
+        verb = "are"
+    else:
+        suffix = ""
+        verb   = "is"
+    logger.info("\nThe following certificate{0} {1} selected for deletion:\n"
+        .format(suffix, verb))
+    for certname in certnames:
+        logger.info("  * " + certname)
+    if not disp.yesno("Are you sure to delete the above certificate{0}?".
+        format(suffix), force_interactive=True):
```

Comment:
The ability to non-interactively delete certificates needs to be preserved. For example, when Certbot is being scripted, this errors unconditionally:

    $ certbot -n delete --cert-name localhost
    Saving debug log to /var/log/letsencrypt/letsencrypt.log

    The following certificate is selected for deletion:

      * localhost
    Missing command line flag or config entry for this setting:
    Are you sure to delete the above certificate?

Maybe `force_interactive=False, default="Yes"`?

Reply:
Hmm, I thought I looked at that. I definitely looked at `cli_flag` where I also seemed to remember the existance of some kind of `--yes-to-all` flag.. But I could also be mistaken with another piece of software :stuck_out_tongue: 

I'll try `default="Yes"`.. It might perhaps also fix the integration test failure.. `tox -e py3-cover` ran perfectly locally though.